python/test/gluon/test_frontend.py

```diff
@@ -23,6 +24,7 @@
 HOPPER_TARGET = GPUTarget("cuda", 90, 32)
 AMPERE_TARGET = GPUTarget("cuda", 80, 32)
 HIP_TARGET = GPUTarget("hip", "gfx1200", 32)
+HIP_TARGET_CDNA3 = GPUTarget("hip", "gfx950", 64)
```

Comment:
`gfx950` is CDNA4.. You can rename the current `HIP_TARGET` as `RDNA4_TARGET` to be consistent with existing target definitions and we also want to have `CDNA3_TARGET` for `gfx942`.